generators/language-node-express/index.js

```diff
@@ -92,6 +94,7 @@ module.exports = class extends Generator {
 	}
 
 	_addInstrumentation(options){
+		this.hasInstrumentation = true;
```

Comment:
we are we not adding this to `swift` and `go` too?

Reply:
I don't know where `swift` and `go` are writing the instrumentation code or how that's generated for those languages